src/app/features/add-network/add-network.tsx

```diff
@@ -70,14 +70,14 @@ export function AddNetwork() {
 
   const setStacksUrl = useCallback(
     (value: string) => {
-      setFieldValue('stacksUrl', value);
+      void setFieldValue('stacksUrl', value);
```

Comment:
I think I was not thinking and used `await` here often to fix the warning. I'll fix in a PR and change to `void`.

Reply:
Thanks! I set it to `void` as I had to fix a bug after some of the stacks UI changes we made where we changed code like this to be `async` to appease eslint but then the function wasn't running